dcql/src/u-dcql.ts

```diff
@@ -121,8 +121,30 @@ export type JsonRecord = v.InferOutput<typeof vJsonRecord>
 export const vStringToJson = v.rawTransform<string, Json>(({ dataset, addIssue, NEVER }) => {
   try {
     return JSON.parse(dataset.value) as Json
-  } catch (error) {
+  } catch (_error) {
     addIssue({ message: 'Invalid JSON' })
     return NEVER
   }
 })
+
+/**
+ * Helper function to provide a custom required message for an object property.
+ *
+ * The behavior was changed in newer valibot versions.
+ *
+ * @see https://github.com/fabian-hiller/valibot/issues/1034
+ */
+export function vCustomRequiredMessage<TSchema extends v.GenericSchema<unknown>>(
+  schema: TSchema,
+  message?: v.ErrorMessage<v.InferIssue<TSchema>>
+) {
+  const outputSchema = v.pipe(
+    v.optional(schema, () => undefined),
+    schema
+  )
+  if (message) {
+    // @ts-expect-error
```

Comment:
Could you add a description to why this is fine? I think for linting stuff I am okay with `no explanation` but typescript errors are a bit more important and a description would help a lot.